ubuntu-advantage

```diff
@@ -85,12 +95,12 @@ install_package_if_missing_file() {
 # Write an APT sources file
 write_apt_list_file() {
     local repo_file="$1"
-    local repo_url="$2"
-    local credentials="$3"
 
+    local url
+    url=$(private_repo_url "$2" "$3")
```

Comment:
What is $2 and $3 now? I kind of like the idea of local meaningful variables, because then we probably don't need docstrings ;)

Reply:
good point :) fixed